composer.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "laminas/laminas-hydrator",
-    "description": "Serialize objects to arrays, and vice versa",
+    "description": "laminas-hydrator provides functionality for hydrating objects and extracting data from them. The component contains concrete implementations for a number of common use cases and provides interfaces for creating custom implementations.",
```

Comment:
Please revert this. We can find a shorter description in another pull request and add this sentence to the introduction also in another pull request.